.github/workflows/ubuntu-ci-x86_64-intel.yaml

```diff
@@ -32,20 +32,25 @@ jobs:
         with:
           submodules: true
 
-      - name: create-env
+      - name: create-buildcache
         run: |
+          # Get day of week to decide whether to use build caches or not
```

Comment:
I adopted this logic from the previously removed workflow on the AWS ParallelCluster (R&D cluster)